### PR TITLE
[BugFix] Fix some unload issues

### PR DESCRIPTION
**Pull request by letian-jiang**
- Disable lzo compression since arrow does not support yet
- Write local timestamp instead of UTC-normalized timestamp for now

## What type of PR is this:

- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: be/src/formats/parquet/file_writer.cpp [hunk 1]
```diff
@@ -231,8 +231,10 @@ arrow::Result<::parquet::schema::NodePtr> ParquetBuildHelper::_make_schema_node(
                                                       ::parquet::Type::INT32, -1, file_column_id.field_id);
     }
     case TYPE_DATETIME: {
+        // TODO(letian-jiang): set isAdjustedToUTC to true, and normalize datetime values
         return ::parquet::schema::PrimitiveNode::Make(
-                name, rep_type, ::parquet::LogicalType::Timestamp(true, ::parquet::LogicalType::TimeUnit::unit::MILLIS),
+                name, rep_type,
+                ::parquet::LogicalType::Timestamp(false, ::parquet::LogicalType::TimeUnit::unit::MILLIS),
```
Comment (Smith-Cruise) [inline be/src/formats/parquet/file_writer.cpp:237]: why need false here

**Review by Smith-Cruise — APPROVED** [review]

File: be/src/formats/parquet/file_writer.cpp (hunk 1 above)
Comment (letian-jiang) [reply be/src/formats/parquet/file_writer.cpp:237]: This is to write un-normalized timestamps. I left a TODO for writing utc-normalized timestamps.

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:heart_eyes: **pass** : 3 / 3 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/formats/parquet/level_builder.cpp|2|2|100.00%|[]|
|:large_blue_circle:|src/formats/parquet/file_writer.cpp|1|1|100.00%|[]|

**Review by dirtysalt — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:heart_eyes: **pass** : 3 / 3 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/formats/parquet/level_builder.cpp|2|2|100.00%|[]|
|:large_blue_circle:|src/formats/parquet/file_writer.cpp|1|1|100.00%|[]|

**Review by dirtysalt — APPROVED** [review]

**Review by stephen-shelby — APPROVED** [review]

**Comment by letian-jiang** [comment]
@Mergifyio backport branch-3.2

**Pull request merged**